tools/IncrementVersion.ps1

```diff
@@ -0,0 +1,58 @@
+$versionPath = "$PSScriptRoot\CustomMSBuild\Versioning.props"
+
+$lastReleaseCommit = git log -n 1 --pretty=format:%H -- $versionPath
```

Comment:
It looks good so far, and we should make sure there's only version changing in this file.